### DISCUSSION TITLE
'Datasource connection error' when trying to update RubyGems

### DISCUSSION THREAD

**Discussion by JrobT** [renovatebot/renovate#15451] [Request Help] [answered]
redacted

**Comment by rarkins** [comment, marked as answer]
Which type of gem server are you using and has it documented which API it supports? I recall there's some servers like gem stash or gem fury with incomplete support from us

**Reply by rarkins** [reply]
Adding documentation URL here: https://help.sonatype.com/repomanager2/ruby%2C-rubygems-and-gem-repositories

It doesn't state exactly what API it supports however it seems Bundler itself works with it fine, so therefore so should Renovate (in future).

First though we need to work out which API bundler supports which we don't - doesn't seem to be the documented api v1. There's been past discussions on gem stash and gem fury so I'm guessing the problems are all related.

Please feel free to create a feature request for "Support Nexus registry for Rubygems" so that it doesn't get lost in discussions.
